### PR TITLE
forgot a data loader

### PR DESCRIPTION
**Pull request by jameschristopher**
Who ever merges this should squash the commits. @twiecki can you take a look?

### CODE REVIEW

File: empyrical/utils.py [hunk 1]
```diff
@@ -396,3 +396,49 @@ def get_symbol_returns_from_yahoo(symbol, start=None, end=None):
...
+    end = get_utc_timestamp(end)
+
+    if symbol == 'SPY':
+        filepath = data_path('spy.csv')
+        rets = get_returns_cached(filepath,
+                                  get_symbol_returns_from_yahoo,
+                                  end,
+                                  symbol='SPY',
+                                  start='1/1/1970',
+                                  end=datetime.now())
```
Comment (georgh0021) [inline empyrical/utils.py]: flake8 is raising the error below. `import datetime` should fix it. This is a new dependency @richafrank will want to know about.

```
$ flake8 .
./empyrical/utils.py:439:39: F821 undefined name 'datetime'
The command "flake8 ." failed and exited with 1 during .
```

Comment (twiecki) [inline empyrical/utils.py]: I don't see how this is a flake8 issue?

Comment (georgh0021) [inline empyrical/utils.py]: flake8 raises an error if you use a package you haven't imported

**Comment by jameschristopher** [comment]
this is ready to be merged btw

**Pull request merged**